tests/blockchain/eth/entities/deployers/test_user_escrow_deployer.py

```diff
@@ -22,61 +22,30 @@
                                                UserEscrowProxyDeployer,
                                                LibraryLinkerDeployer)
 from nucypher.crypto.api import keccak_digest
-from nucypher.utilities.sandbox.constants import USER_ESCROW_PROXY_DEPLOYMENT_SECRET
-
+from nucypher.utilities.sandbox.constants import USER_ESCROW_PROXY_DEPLOYMENT_SECRET, INSECURE_DEVELOPMENT_PASSWORD
 
 user_escrow_contracts = list()
 NUMBER_OF_PREALLOCATIONS = 50
 
 
-@pytest.fixture(scope='module')
-def user_escrow_proxy_deployer(testerchain, session_agency, test_registry):
-    print("ENTER USER ESCROW")
-    testerchain = testerchain
-    deployer = testerchain.etherbase_account
-    user_escrow_proxy_deployer = UserEscrowProxyDeployer(deployer_address=deployer,
-                                                         registry=test_registry)
-    return user_escrow_proxy_deployer
-
-
 @pytest.mark.slow()
 def test_user_escrow_deployer(testerchain,
-                              session_agency,
-                              user_escrow_proxy_deployer,
+                              agency,
                               deployment_progress,
                               test_registry):
-    testerchain = testerchain
-    deployer_account = testerchain.etherbase_account
-    secret_hash = keccak_digest(USER_ESCROW_PROXY_DEPLOYMENT_SECRET.encode())
-
-    user_escrow_proxy_receipts = user_escrow_proxy_deployer.deploy(secret_hash=secret_hash,
-                                                                   progress=deployment_progress)
-
-    # deployment steps must match expected number of steps
-    assert deployment_progress.num_steps == len(user_escrow_proxy_deployer.deployment_steps) == 2
-    assert len(user_escrow_proxy_receipts) == 2
-
-    for step in user_escrow_proxy_deployer.deployment_steps:
-        assert user_escrow_proxy_receipts[step]['status'] == 1
-
-    deployer = UserEscrowDeployer(deployer_address=deployer_account,
-                                  registry=test_registry)
 
+    deployer = UserEscrowDeployer(deployer_address=testerchain.etherbase_account, registry=test_registry)
```

Comment:
This changeset is getting rid of the original intention of this test, which is to check in detail a manual deployment of `UserEscrowProxy`. The test here should be analogous to what's in  `test_policy_manager_deployer`, which using first a `staking_escrow_deployer` fixture, tests a manual deployment of `PolicyManager`. Translating to `UserEscrowProxy` means that the test must use a `staking_escrow_deployer` and `policy_manager_deployer` fixture in order to test manually a deployment of `UserEscrowProxy`.

In other words, IMO these tests shouldn't use the `agency` fixture.

Reply:
Yeah - I Agree.  I must have overlooked this.

Reply:
It's looking better, but still needs to check the details of the `UserEscrowDeployer`(which was the point of the original test). In other words, these lines are missing:
```python
user_escrow_proxy_receipts = user_escrow_proxy_deployer.deploy(secret_hash=secret_hash,
                                                                   progress=deployment_progress)
# deployment steps must match expected number of steps
assert deployment_progress.num_steps == len(user_escrow_proxy_deployer.deployment_steps) == 2
assert len(user_escrow_proxy_receipts) == 2

for step in user_escrow_proxy_deployer.deployment_steps:
        assert user_escrow_proxy_receipts[step]['status'] == 1
```

Reply:
:+1:

Reply:
:heavy_check_mark: